server/models/auth/forgotPassword.js

```diff
@@ -37,7 +37,7 @@ const emailMessage = (otp, host, port) => {
   return `<p>Hello !</p>
           <p>The otp for resetting your password is ${otp}</p>
           <p>Please reset your password by visiting the following link</p>
-          <a href='http://${host}:${port}/auth/reset_password'>Reset password</a>`
+          <a href='http://${host}:${port}/auth/reset-password'>Reset password</a>`
```

Comment:
`reset-password` will break the APIs

Reply:
either we keep it as `reset_password` or we may need to change it in other files to

Reply:
No, it's the URL for frontend. The form for resetting password is made available in the frontend. So, I guess it should not affect the API.

Reply:
Also, do note the changes made above this, in port and host. I have made it process.env.FRONTEND_PORT and process.env.FRONTEND_HOST

Reply:
You can run both the server and the frontend locally and test it.

Reply:
okies...cool!